src/status_im/new_group/handlers.cljs

```diff
@@ -13,6 +13,20 @@
             [taoensso.timbre :as log]
             [status-im.navigation.handlers :as nav]))
 
+(defmethod nav/preload-data! :add-contacts-toggle-list
+  [db _]
+  (->
+    (assoc db :selected-contacts #{})
+    (assoc-in [:toolbar-search :show] nil)
+    (assoc-in [:toolbar-search :text] "")))
+
+(defmethod nav/preload-data! :add-participants-toggle-list
+  [db _]
+  (->
```

Comment:
The next 4 lines could be extracted into a separate function to avoid duplication.